docker_db.sh

```diff
@@ -89,7 +89,7 @@ edb() {
 
 db2() {
     docker rm -f db2 || true
-    docker run --name db2 --privileged -e DB2INSTANCE=orm_test -e DB2INST1_PASSWORD=orm_test -e DBNAME=orm_test -e LICENSE=accept -e AUTOCONFIG=false -e ARCHIVE_LOGS=false -e TO_CREATE_SAMPLEDB=false -e REPODB=false -p 50000:50000 -d ibmcom/db2:11.5.5.0
+    docker run --name db2 --privileged -e DB2INSTANCE=orm_test -e DB2INST1_PASSWORD=orm_test -e DBNAME=orm_test -e LICENSE=accept -e PERSISTENT_HOME=false -e AUTOCONFIG=false -e ARCHIVE_LOGS=false -e TO_CREATE_SAMPLEDB=false -e REPODB=false -p 50000:50000 -d ibmcom/db2:11.5.7.0
```

Comment:
According to https://hub.docker.com/r/ibmcom/db2 `PERSISTENT_HOME` should only be specified on windows. Maybe that's the issue?

Reply:
I've tested PERSISTENT_HOME=false locally (on linux) and all it does is skip the mount points... locally it helps as then the filesystem is cached in memory.

But sure we can try..

Reply:
pushed again with that change omitted

Reply:
Seems to work and it is really fast now :)

Reply:
OK but it's suspicious... before merging I'll run it a couple more times to ensure it's not flaky